### PR TITLE
Live activities update 3 2 23

### PR DESCRIPTION
**Pull request by josh-mccrowell-braze**

### CODE REVIEW

File: _docs/_api/endpoints/messaging/live_activity/update.md [hunk 1]
```diff
@@ -32,9 +35,9 @@ Before using this endpoint, you must register an activity with the Braze Swift S
...
    "end_activity": "(optional, boolean) If true, this request ends the Live Activity.",
-   "dismissal_date": "(optional, datetime in ISO-8601 format) the time to remove the Live Activity from the user’s UI. If this time is in the past, the Live Activity will be removed immediately.",
-   "stale_date": "(optional, datetime in ISO-8601 format) when the Live Activity content is marked as outdated in the user’s UI.",
-   "notification": "(optional, object ) include an apple_push object to define a push notification that will also be displayed with this update."
+   "dismissal_date": "(optional, datetime in ISO-8601 format) The time to remove the Live Activity from the user’s UI. If this time is in the past, the Live Activity will be removed immediately.",
+   "stale_date": "(optional, datetime in ISO-8601 format) The time the Live Activity content is marked as outdated in the user’s UI.",
+   "notification": "(optional, object ) Include an `apple_push` object to define a push notification that creates an alert for the user."
```
Comment (jerielng) [inline _docs/_api/endpoints/messaging/live_activity/update.md:40]: Based on the test pushes we've sent, should this be:
```suggestion
   "notification": "(optional, object ) Include an `alert` object to define a push notification that creates an alert for the user."
```

@christorbraze for sanity check

Comment (josh-mccrowell-braze) [reply _docs/_api/endpoints/messaging/live_activity/update.md:40]: Isn't `alert` a property of an `apple_push` object?

File: _docs/_api/endpoints/messaging/live_activity/update.md [hunk 2]
```diff
@@ -48,7 +51,7 @@ Before using this endpoint, you must register an activity with the Braze Swift S
...
-| `notification` | Optional | Object | Include an [`apple_push`]({{site.baseurl}}/api/objects_filters/messaging/apple_object/) object to define a push notification that will also be displayed with this update. |
+| `notification` | Optional | Object | Include an [`apple_push`]({{site.baseurl}}/api/objects_filters/messaging/apple_object/) object to define a push notification. This behavior of this push notification depends on if the user is active or if the user is using a proxy device. {::nomarkdown}<ul><li>If a <code>notification</code> is included and the user is active on their iPhone when the update is delivered, the updated Live Activity UI will slide down and display like a push notification.</li><li>If a <code>notification</code> is included and the user is not active on their iPhone, their screen will light up to display the updated Live Activity UI on their lock screen.</li><li>The <code>notification alert</code> will not display as a standard push notification. Instead, if a user has a proxy device, like an Apple Watch, the <code>alert</code> will be displayed there.</li></ul>{:/} |
```
Comment (jerielng) [inline _docs/_api/endpoints/messaging/live_activity/update.md:54]:
```suggestion
| `notification` | Optional | Object | Include an [`apple_push`]({{site.baseurl}}/api/objects_filters/messaging/apple_object/) object to define a push notification. This behavior of this push notification depends on if the user is active or if the user is using a proxy device. {::nomarkdown}<ul><li>If a <code>notification</code> is included and the user is active on their iPhone when the update is delivered, the updated Live Activity UI will slide down and display like a push notification.</li><li>If a <code>notification</code> is included and the user is not active on their iPhone, their screen will light up to display the updated Live Activity UI on their lock screen.</li><li>The <code>notification alert</code> will not display as a standard push notification. Additionally, if a user has a proxy device, like an Apple Watch, the <code>alert</code> will be displayed there.</li></ul>{:/} |
```

For context, if I have an Apple Watch connected to my phone and I'm not active when the push comes through, I won't receive the ping on my phone, only on my Watch.

**Review by jerielng — APPROVED** [review]
Couple of questions, but the rest LGTM! The screenshots look awesome!!

File: _docs/_api/endpoints/messaging/live_activity/update.md (hunk 1 above)
Comment (christorbraze) [reply _docs/_api/endpoints/messaging/live_activity/update.md:40]: Yes, that's right ([docs link](https://www.braze.com/docs/api/objects_filters/messaging/apple_object/))

Comment (josh-mccrowell-braze) [reply _docs/_api/endpoints/messaging/live_activity/update.md:40]: Thanks both!

**Review by KellieHawks — APPROVED** [review]

**Pull request merged**